tad/loaders/data_loader.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from pathlib import Path
+
+import joblib
+import pandas as pd
+from pandas import DataFrame
+from pandas.io.parsers import TextFileReader
+
+
+class DataLoader:
+    """
+    DataLoader class offers methods to load supported data formats
+    """
+
+    supported_data_extensions: [] = [".csv", ".sav"]
```

Comment:
As discussed, the other types may be added later. The reason to support the current format(s) is because I had testcases for these.

I have used the Enum sollution as suggested by Christopher; makes the code a lot more readable, thanks.